### PR TITLE
Fix main readme to be consistent with main repo

### PR DESCRIPTION
**Pull request by cijothomas**
Fixes #
Design discussion issue (if applicable) #

## Changes

Please provide a brief description of the changes here.

## Merge requirement checklist

* [ ] [CONTRIBUTING](https://github.com/open-telemetry/opentelemetry-rust/blob/main/CONTRIBUTING.md) guidelines followed
* [ ] Unit tests added/updated (if applicable)
* [ ] Appropriate `CHANGELOG.md` files updated for non-trivial, user-facing changes
* [ ] Changes in public API reviewed (if applicable)

### CODE REVIEW

**Review by lalitb — APPROVED** [review]
Thanks for cleanup

**Review by hdost — CHANGES_REQUESTED** [review]
I think it looks good, just the suggested commits lalit had.

File: README.md [hunk 1]
```diff
@@ -13,95 +9,21 @@ The Rust [OpenTelemetry](https://opentelemetry.io/) implementation contrib compo
...
-[`actix-web`]: https://crates.io/crates/actix-web
-[`opentelemetry-application-insights`]: https://crates.io/crates/opentelemetry-application-insights
-[Azure Application Insights]: https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview
-[`opentelemetry-tide`]: https://crates.io/crates/opentelemetry-tide
-[`Tide`]: https://crates.io/crates/tide
-[`opentelemetry-stackdriver`]: https://crates.io/crates/opentelemetry-stackdriver
-[Cloud Trace]: https://cloud.google.com/trace/
+Check individual folders for usage guidelines and examples.
 
 ## Supported Rust Versions
```
[the remark this answers is not in the document]
Comment (cijothomas) [reply README.md:28]: keeping this now, will revisit to have better wording here.

**Review by lalitb — APPROVED** [review]

**Comment by lalitb** [comment]
LGTM with the updates. Thanks.

**Comment by cijothomas** [comment]
@hdost Please share any additional feedback and will address them in subsequent PRs. I dismissed review to unblock merge.

**Pull request merged**